### PR TITLE
Erdjs interaction examples

### PR DESCRIPTION
**Pull request by claudiu725**
Added interaction examples for several example smart contracts, as well as for basic ESDT scenarios.
The `erdjs` contract wrapper implementation which enables these examples can be found in: https://github.com/ElrondNetwork/elrond-sdk-erdjs/pull/9

### CODE REVIEW

File: contracts/examples/crowdfunding-esdt/interaction/Crowdfunding-egld.erdjs.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+// Store alice's current balance (we'll use this to check the balance difference later on)
+let aliceBalanceBefore = await erdSys.getBalance(alice, Egld);
+erdjs.print(aliceBalanceBefore);
+
+// Wait a minute first, otherwise you'll get the "cannot claim before deadline" error
+// If the claim doesn't return an error - there are two possibilities:
+// - the funding failed, and 1.5 EGLD are sent back to both bob and carol
+// - it was succesful and alice receives 3 EGLD
+// Because the target sum specified on deployment was 2 EGLD, and we have 3 EGLD, the funding should be succesful
```
Comment (alyn509) [inline contracts/examples/crowdfunding-esdt/interaction/Crowdfunding-egld.erdjs.md:41]: I think you should make a separate section in the docs where you write these stuff..

File: contracts/examples/crowdfunding-esdt/interaction/Crowdfunding-esdt.erdjs.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+// Store alice's current balance (we'll use this to check the balance difference later on)
+let aliceBalanceBefore = await erdSys.getBalance(alice, MyToken);
+erdjs.print(aliceBalanceBefore);
+
+// Wait a minute first, otherwise you'll get the "cannot claim before deadline" error
+// If the claim doesn't return an error - there are two possibilities:
+// - the funding failed, and 1.5 MYTOKEN are sent back to both bob and carol
+// - it was succesful and alice receives 3 MYTOKEN
+// Because the target sum specified on deployment was 2 MYTOKEN, and we have 3 MYTOKEN, the funding should be succesful
```
Comment (alyn509) [inline contracts/examples/crowdfunding-esdt/interaction/Crowdfunding-esdt.erdjs.md:49]: same as above

File: contracts/examples/multisig/interaction/Multisig-adder-egld.erdjs.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+First [set up a node terminal](../../../../tutorial/src/interaction/interaction-basic.md).
+
+```javascript
+let erdjs = await require('@elrondnetwork/erdjs');
+let { erdSys, Egld, wallets: { alice, bob, carol }} = await erdjs.setupInteractive("local-testnet");
+
+let multisig = await erdSys.loadWrapper("contracts/examples/multisig");
+
+// Deploy a multisig contract with a quorum of 3, but 4 possible signers: alice, bob, carol, dan
+// A quorum is the minimum required number of signers in order to be able to perform a proposed action
```
Comment (alyn509) [inline contracts/examples/multisig/interaction/Multisig-adder-egld.erdjs.md:12]: new notion => write it in the docs

File: contracts/examples/crowdfunding-esdt/interaction/Crowdfunding-esdt.erdjs.md (hunk 2 above)
Comment (claudiu725) [reply contracts/examples/crowdfunding-esdt/interaction/Crowdfunding-esdt.erdjs.md:49]: see above

File: contracts/examples/crowdfunding-esdt/interaction/Crowdfunding-egld.erdjs.md (hunk 1 above)
Comment (claudiu725) [reply contracts/examples/crowdfunding-esdt/interaction/Crowdfunding-egld.erdjs.md:41]: This info depends on the following example

File: contracts/examples/multisig/interaction/Multisig-adder-egld.erdjs.md (hunk 3 above)
Comment (claudiu725) [reply contracts/examples/multisig/interaction/Multisig-adder-egld.erdjs.md]: This info is already in the readme for this contract. I left a note at the top of this script pointing to that readme.

**Review by alyn509 — APPROVED** [review]

**Pull request merged**